rdkit/Chem/Randomize.py

```diff
@@ -8,64 +8,30 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
+import random
 from rdkit import Chem
-from rdkit import RDRandom as random
 
 
-def RandomizeMolBlock(molB):
-  splitB = molB.split('\n')
-  res = []
-  res.extend(splitB[0:3])
-  idx = 3
-  inL = splitB[idx]
-  res.append(inL)
-  nAts = int(inL[0:3])
-  nBonds = int(inL[3:6])
+def _shuffle(indices: list[int]) -> list[int]:
```

Comment:
We don't use type annotations in the RDKit, please remove these

Reply:
See [e929aa7](https://github.com/rdkit/rdkit/pull/7232/commits/e929aa7ca89b496d7b7600de651b44b47e35ee76).
CI seems to be failing because of the type annotations.